.aws/deploy/backend-task-definition.prod.json

```diff
@@ -0,0 +1,221 @@
+        },
+        {
+          "name": "ISOMERPAGES_REPO_PAGE_COUNT",
+          "valueFrom": "PROD_ISOMERPAGES_REPO_PAGE_COUNT"
+        },
+        { "name": "JWT_SECRET", "valueFrom": "PROD_JWT_SECRET" },
+        {
+          "name": "MAX_NUM_OTP_ATTEMPTS",
+          "valueFrom": "PROD_MAX_NUM_OTP_ATTEMPTS"
+        },
+        {
+          "name": "MOCK_AMPLIFY_DOMAIN_ASSOCIATION_CALLS",
+          "valueFrom": "PROD_MOCK_AMPLIFY_DOMAIN_ASSOCIATION_CALLS"
+        },
+        { "name": "MUTEX_TABLE_NAME", "valueFrom": "PROD_MUTEX_TABLE_NAME" },
+        {
+          "name": "NETLIFY_ACCESS_TOKEN",
+          "valueFrom": "PROD_NETLIFY_ACCESS_TOKEN"
+        },
+        { "name": "NODE_ENV", "valueFrom": "PROD_NODE_ENV" },
+        { "name": "OTP_EXPIRY", "valueFrom": "PROD_OTP_EXPIRY" },
+        { "name": "OTP_SECRET", "valueFrom": "PROD_OTP_SECRET" },
+        {
+          "name": "OUTGOING_QUEUE_URL",
+          "valueFrom": "PROD_OUTGOING_QUEUE_URL"
+        },
+        { "name": "POSTMAN_API_KEY", "valueFrom": "PROD_POSTMAN_API_KEY" },
+        {
+          "name": "POSTMAN_SMS_CRED_NAME",
+          "valueFrom": "PROD_POSTMAN_SMS_CRED_NAME"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/aws/elasticbeanstalk/cms-backend-prod-node18/var/log/web.stdout.log",
+          "awslogs-region": "ap-southeast-1",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    },
+    {
+      "name": "dd-agent",
+      "image": "public.ecr.aws/datadog/agent:latest",
+      "portMappings": [
+        {
+          "containerPort": 8126,
+          "hostPort": 8126,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [
+        {
+          "name": "TZ",
+          "value": "Asia/Singapore"
+        },
+        {
+          "name": "DD_APM_NON_LOCAL_TRAFFIC",
+          "value": "true"
+        },
+        {
+          "name": "ECS_FARGATE",
+          "value": "true"
+        },
+        {
+          "name": "DD_APM_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.com"
+        },
+        {
+          "name": "DD_ENV",
+          "value": "prod"
+        },
+        {
+          "name": "DD_SERVICE",
+          "value": "isomer"
+        },
+        {
+          "name": "DD_TAGS",
+          "value": "service:isomer"
+        },
+        {
+          "name": "DD_AGENT_MAJOR_VERSION",
+          "value": 7
+        },
+        {
+          "name": "DD_LOGS_INJECTION",
+          "value": true
+        },
+        {
+          "name": "DD_TRACE_STARTUP_LOGS",
+          "value": true
+        },
+        {
+          "name": "DD_API_KEY",
+          "valueFrom": "<DD_API_KEY>"
+        }
+      ],
+      "mountPoints": [],
+      "volumesFrom": [],
+      "secrets": [],
+      "logConfiguration": {
```

Comment:
logs are directly piped to DD from now? no more cloudwatch?

Reply:
iirc is cw -> dd, please correct me if i am wrong

Reply:
this is piped to a cloudwatch log group which we then forward to dd

Reply:
okay